From 548964307422267e2513d83352de0a882947ffd1 Mon Sep 17 00:00:00 2001
From: "John Lawrence M. Penafiel" <penafieljlm@gmail.com>
Date: Sat, 9 Feb 2019 03:33:18 +0800
Subject: [PATCH] Added rootwait to CONFIG_MENDER_BOOTCOMMAND

---
 include/env_mender.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/env_mender.h b/include/env_mender.h
index 705c39e..1ef6485 100644
--- a/include/env_mender.h
+++ b/include/env_mender.h
@@ -89,7 +89,7 @@
 
 #define CONFIG_MENDER_BOOTCOMMAND                                       \
     "run mender_setup; "                                                \
-    "setenv bootargs root=${mender_kernel_root} ${bootargs}; "          \
+    "setenv bootargs root=${mender_kernel_root} rootwait ${bootargs}; "          \
     "if test \"${fdt_addr_r}\" != \"\"; then "                          \
     "load ${mender_uboot_root} ${fdt_addr_r} /boot/${mender_dtb_name}; " \
     "fi; "                                                              \
-- 
2.9.0

